### PR TITLE
Note about escaping quotes in SQL Server advanced queries

### PR DESCRIPTION
**Pull request by pivnicek**
Fixes #ISSUE_NUMBER

Proposed changes:

-  Add a note about escaping quotes in SQL Server advanced queries

### CODE REVIEW

**Comment by pivnicek** [comment]
Hmm,  I'm not really happy with it, it's too much text now.  What do you think @ujovlado ?

**Comment by pivnicek** [comment]
Add link to not yet released docs https://github.com/keboola/connection-docs/pull/95

**Comment by pivnicek** [comment]
Ok @ujovlado the documentation is live.  Can you check it please?

File: src/scripts/modules/ex-db-generic/react/components/QueryEditor.jsx [hunk 1]
```diff
@@ -495,9 +495,7 @@ export default React.createClass({
     const helpText = getQueryEditorHelpText(this.props.componentId);
     if (helpText) {
       return (
-        <div className="help-block">
-          {helpText}
-        </div>
+        <div className="help-block" dangerouslySetInnerHTML={{__html: helpText}}/>
```
Comment (ujovlado) [inline src/scripts/modules/ex-db-generic/react/components/QueryEditor.jsx]: It is called `dangerouslySetInnerHTML` because we shouldn't use it ...

Comment (pivnicek) [reply src/scripts/modules/ex-db-generic/react/components/QueryEditor.jsx]: yeah, I know, sorry.  brutal hack that I should have expected to be rejected.  I'll just say I was testing you :-D.  Will have an update shortly

**Comment by pivnicek** [comment]
ok, I removed the hack @ujovlado , sorry about that

**Pull request merged**